### PR TITLE
Add matchers

### PR DESCRIPTION
**Pull request by wachri**

### CODE REVIEW

File: lib/jasmine.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2018 Hetzner Cloud GmbH
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+declare namespace jasmine {
+    interface Matchers<T> {
+        toBePresent(): boolean;
```
Comment (iMarv) [inline lib/jasmine.ts]: Please add documentation about the difference between being `present` and being `displayed`

File: test/async/matcher.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        it('should expect css selector string not to be present', async () => {
+            await expect('.xe2easdfee').not.toBeDisplayed();
+        });
+
+        it('should expect element not to be displayed', async () => {
+            await waitToBeNotDisplayed(waitsPage.waitToBeNotDisplayed);
+            await expect(waitsPage.waitToBeNotDisplayed).not.toBeDisplayed();
+        });
+
+        it('should expect element not to be when it is only present', async () => {
```
Comment (iMarv) [inline test/async/matcher.e2e-spec.ts]: expect element not to be what?

File: test/async/matcher.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        it('should expect css selector string not to be present', async () => {
+            await expect('.xe2easdfee').not.toBePresent();
+        });
+
+        it('should expect element not to be present', async () => {
+            await waitToBeNotDisplayed(waitsPage.waitToBeNotPresent);
+            await expect(waitsPage.waitToBeNotPresent).not.toBePresent();
+        });
+
+        it('should expect element to be displayed', async () => {
```
Comment (iMarv) [inline test/async/matcher.e2e-spec.ts]: Confusing testname.
expecting the element to be displayed, but using `.toBePresent()` in the test

File: test/flow/matcher.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
+/**
```
Comment (iMarv) [inline test/flow/matcher.e2e-spec.ts]: As the test are copied, the comments from the async tests apply here aswell

File: dist/matchers.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+"use strict";
```
[the remark this answers is not in the document]
Comment (wachri) [reply dist/matchers.js]: As discussed for the moment I let it in

**Pull request merged**